docs/src/enterprise/getstarted/migrate-from-fluffy.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Migrate from Fluffy to lakeFS Enterprise
+description: How to Migrate from a Fluffy-Based Enterprise Version to New lakeFS Enterprise
+---
+
+# Migrate From Fluffy to lakeFS Enterprise
+
+The new lakeFS Enterprise integrates all enterprise features directly into a single binary, eliminating the need for the separate Fluffy service. This simplifies deployment, configuration, and maintenance.
+
+## Pre-requisite
+
+1. you posses a lakeFS Enterprise license.
+1. You're using lakeFS enterprise binary or the image in Dockerhub treeverse/lakefs-enterprise with fluffy.
+<!-- TODO: insert lakefs version  -->
+1. Your lakeFS-Enterprise version is >= x.y.z. 
```

Comment:
pre-requisites order is %2 
1. then 2. then 1.

Reply:
Now it's 1 then 1 then 2 🙄 
![image](https://github.com/user-attachments/assets/4bee68d1-5c0b-4730-9718-bb6e0e7022ac)